server/odc-service/src/main/java/com/oceanbase/odc/service/regulation/risklevel/RiskLevelService.java

```diff
@@ -117,6 +117,11 @@ public Optional<RiskLevel> findById(@NonNull Long id) {
         return riskLevelRepository.findById(id).map(this::entityToModel);
     }
 
+    @SkipAuthorize("internal usage")
+    public Optional<RiskLevel> findRawById(@NonNull Long id) {
```

Comment:
How should we understand `Raw` here in your method name?

Reply:
I deleted this method and use Repository instead.